geode-core/src/integrationTest/java/org/apache/geode/cache/query/functional/INOperatorJUnitTest.java

```diff
@@ -225,11 +225,11 @@ public void testUNDEFINED() throws Exception {
 
     q = CacheUtils.getQueryService().newQuery(" UNDEFINED IN SET(UNDEFINED)");
     result = q.execute();
-    assertThat(result).isEqualTo(QueryService.UNDEFINED);
+    assertThat(result).isEqualTo(TRUE);
```

Comment:
Another syntax to consider: `assertThat(result).isTrue();`

Reply:
+1 for consistency.